### PR TITLE
Docs: update diagnostic-tools trust con

### PR DESCRIPTION
**Pull request by KipsasJaujoj**
Please provide a description for what this PR is for.

I'm going through multi cluster routing troubleshooting documentation and noticed there is some different secret / path names.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
@istio/wg-environments-maintainers Would you verify? Thanks.

**Review by howardjohn — COMMENTED** [review]
cc @jaellio

File: content/en/docs/ops/diagnostic-tools/multicluster/index.md [hunk 1]
```diff
@@ -121,8 +121,8 @@ To verify certs are configured correctly, you can compare the root-cert in each
 
 {{< text bash >}}
 $ diff \
-   <(kubectl --context="${CTX_CLUSTER1}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}') \
-   <(kubectl --context="${CTX_CLUSTER2}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}')
+   <(kubectl --context="${CTX_CLUSTER1}" -n istio-system get secret istio-ca-secret -ojsonpath='{.data.ca-cert\.pem}') \
+   <(kubectl --context="${CTX_CLUSTER2}" -n istio-system get secret istio-ca-secret -ojsonpath='{.data.ca-cert\.pem}')
```
Comment (frankbu) [inline content/en/docs/ops/diagnostic-tools/multicluster/index.md:125]: This change does not look right. The current names (`cacerts` and `root-cert.pem`) are exactly the ones used in the referenced document used for setting up certs for the multicluster installation docs. https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/#plug-in-certificates-and-key-into-the-cluster

Comment (KipsasJaujoj) [reply content/en/docs/ops/diagnostic-tools/multicluster/index.md:125]: I'm going through this at the moment to get multi cluster routing working. But there were no such secrets in my istio deployments and no such keys. I'm working with version `19.1`. 
I was using helm to install.
If it happens these secrets appear, I'll update this PR with note about them not being present.

Comment (frankbu) [reply content/en/docs/ops/diagnostic-tools/multicluster/index.md]: The multicluster installation before-you-begin section creates the secret: https://istio.io/latest/docs/setup/install/multicluster/before-you-begin/#configure-trust

There is a tip that you can avoid this if you're just deploying a single control plane (simple primary-remote setup), but none of this applies in that case anyway. This is dealing with the general case of multiple primary clusters where a shared root of trust is needed.

File: content/en/docs/ops/diagnostic-tools/multicluster/index.md [hunk 2]
```diff
@@ -125,8 +125,8 @@ $ diff \
    <(kubectl --context="${CTX_CLUSTER2}" -n istio-system get secret cacerts -ojsonpath='{.data.root-cert\.pem}')
 {{< /text >}}
 
-You can follow the [Plugin CA Certs](/docs/tasks/security/cert-management/plugin-ca-cert/) guide, ensuring to run
-the steps for every cluster.
+If root-certs does not match or secrets does not exist at all, you can follow the [Plugin CA Certs](/docs/tasks/security/cert-management/plugin-ca-cert/)
```
Comment (jaellio) [inline content/en/docs/ops/diagnostic-tools/multicluster/index.md:128]: nit: 
```suggestion
If the root-certs do not match or the secret does not exist at all, you can follow the [Plugin CA Certs](/docs/tasks/security/cert-management/plugin-ca-cert/)
```

**Review by craigbox — APPROVED** [review]
I think that with this clarifying change, we're good to merge the PR.  @KipsasJaujoj, if you think that any more detail would be worthwhile, please feel free to open another. Thanks!

**Pull request merged**